cmd/http/main.go

```diff
@@ -37,6 +36,7 @@ func main() {
 	// its implementation https://github.com/golang/go/blob/master/src/math/rand/rng.go#L25
 	rand.Seed(time.Now().UnixNano())
 
+	url := flag.String("url", ":8090", "proxy url")
```

Comment:
understand it's from before this PR. What does proxy refer to here?

Reply:
to my understanding this does not have direct relationship to proxy but the rkv service endpoint url. will change the description accordingly.

Reply:
It is the http proxy for the rkv. 

rkv itself might run on grpc if we have multiple instances running in different regions.  Since there is only one instance running by design in the initial phase. HTTP proxy will not dispatch request among rkv instances.

Reply:
ah i see. thanks.